multilog/sink.go

```diff
@@ -70,6 +70,8 @@ func (slog *sinkLog) Pour(ctx context.Context, v interface{}) error {
 }
 
 // Close closes the root-log..? seems wrong
+// rational here beeing having multiple mlogs on one root (?)
```

Comment:
The code that's commented out doesn't close the root log but the multilog, which seems like a reasonable thing to do. But yeah, not sure when this function get's called, and whether the multilog is being read afterwards. Maybe have something like a "multiple-open tracker" that tracks all the users of the multilog and closes it if all users closed it? Not saying this should happen now, but let's at least put a TODO here that we need to figure out how to proceed correctly.